tsp/inc/cut.h

```diff
@@ -34,23 +34,22 @@
 #define CC_MINCUT_BIGDOUBLE (1e30)
 #define CC_MINCUT_ONE_EPSILON (0.000001)
 
-int CCcut_mincut(int ncount, int ecount, int *elist, double *dlen, double *cutval, int **cut, int *cutcount),
```

Comment:
This is an external library, there is any need to comply c-lang formating under `tsp`. This is an external library, so I perhaps is better not to touch it

Reply:
I reverted the changes to that external library.